pkg/index/validation/validate.go

```diff
@@ -79,6 +79,9 @@ func ValidatePlugin(name string, p index.Plugin) error {
 	if p.Spec.ShortDescription == "" {
 		return errors.New("should have a short description")
 	}
+	if strings.ContainsAny(p.Spec.ShortDescription, "\r\n") {
```

Comment:
Do you know if we call this on the parse path?

If so it might cause an issue for people who run on this version of krew with an older index. I am not sure how/if that’d happen. Wdyt?

Reply:
:face_with_head_bandage: You are totally right. This _is_ called in the parse path. However, I think we are fine here:

* the new krew version will only be installed when the index was updated as well. The only way one could break it, would be by purpose, if somebody manually downloads a newer krew version and tries to work with an old index version.
* old krew versions might have a newer index. However that doesn't really matter, because the new index will also be valid for the old krew version.